api/all/src/main/java/io/opentelemetry/api/baggage/propagation/Parser.java

```diff
@@ -18,6 +19,21 @@
  */
 class Parser {
 
+  private static final BitSet EXCLUDED_KEY_CHARS = new BitSet(128);
+  private static final BitSet EXCLUDED_VALUE_CHARS = new BitSet(128);
+
+  static {
+    for (char c :
+        new char[] {
+          '(', ')', '<', '>', '@', ',', ';', ':', '\\', '"', '/', '[', ']', '?', '=', '{', '}'
+        }) {
+      EXCLUDED_KEY_CHARS.set(c);
+    }
+    for (char c : new char[] {'"', ',', ';', '\\'}) {
+      EXCLUDED_VALUE_CHARS.set(c);
+    }
+  }
+
```

Comment:
Since this stuff is only used inside the Element class, what would you think about moving this stuff over there, and having two named creational methods for the Key and Value exclusions? I think that might make this code a little bit easier to understand. Not a blocker for this change, but might be a nice readability improvement.

Reply:
Yeah good idea